tests/quality_pragmaitc/test_refinement.py

```diff
@@ -274,8 +265,8 @@ def test_aGoalOrDecomposedWithTwoTasksMayBeAchievableAtOnlyOneBranch():
     interp.addQualityConstraint(qc)
 
     plan = goal.isAchievable(fullContext, interp)
-    assert plan.getTasks().contains(task2)
-    assert plan.getTasks().contains(task1)
+    assert task2 in plan.getTasks()
+    assert task1 not in plan.getTasks()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4895731)